grunt-ngdocs-index.tmpl

```diff
@@ -89,30 +89,82 @@
   </script>
 </head>
 
-<body>
-  <header class="header">
-    <div class="navbar navbar-fixed-top">
-      <div class="navbar-inner">
-        <div class="container-fluid navbar-inverse">
-          <% if (image) { %>
-            <% if (imageLink) { %>
-              <a href="<%= imageLink %>">
-            <% } %>
-              <img class="pull-left" src="<%= image %>">
-            <% if (imageLink) { %>
+<body class="showcase">
+  <header>
+    <nav class="navbar navbar-default navbar-fixed-top">
+      <div class="container-fluid">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle" ng-click="isOpen = !isOpen">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <div class="navbar-brand">
+            <a id="logo" title="Angular Patternfly" href="#">
+              PATTERNFLY-<span class="secondary-logo" ng-bind-template="<%= title %>"></span>
```

Comment:
Do we need the all caps PATTERNFLY here? Is this the design across all the showcases?

Reply:
Yes @jeff-phillips-18 , this is the uniformed design across all the showcases.

Reply:
So are other components PATTERNFLY - Patternfly NG, and PATTERNFLY - Patternfly Web Components? Seems redundant.

Reply:
@jeff-phillips-18 It should be PATTERNFLY-NG, PATTERNFLY-REACT, PATTERNFLY-WEB COMPONENTS, PATTERNFLY-ANGULAR.  so here <%= title %> the title should be ANGULAR.

Reply:
This repository has been around far too long to start calling it PATTERNFLY-ANGULAR. It has been known as Angular Patternfly and should remain that way.

Reply:
@jeff-phillips-18 yes, we discussed it before. sorry for forgetting it.
so here the name should be ANGULAR PATTERNFLY, does it make sense?